### DISCUSSION TITLE
job info popup does not adjust to description

### DISCUSSION THREAD

**Discussion by BAB-Gaming** [FakeFishGames/Barotrauma#14486] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/1e217f66-7327-49c8-b1c4-236d6f6710ed)

### Reproduction steps

4k res (havn't tested other res)
host server
click on (i) on the job
i have not tried solving this in any way, as it is minor and presumably just an oversight that effects 4k resolution users - (which i have to be one as my monitor becomes discoloured in 1080p ): )

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
